### PR TITLE
Add .NET Client links

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

File: src/docs/admin/references/properties/scheduler.properties [hunk 1]
```diff
@@ -43,7 +43,7 @@ pa.scheduler.home=.
 # Example https://myserver.mydomain.com/catalog
 #pa.catalog.rest.public.url=
 
-# Set this property to define the public url of Service Automation (e.g. if behind a reverse proxy or accessed through a domain)
+# Set this property to define the public url of the cloud automation service (e.g. if behind a reverse proxy or accessed through a domain)
```
Comment (mboussaa) [inline src/docs/admin/references/properties/scheduler.properties:46]: Cloud Automation

File: src/docs/admin/references/properties/scheduler.properties [hunk 2]
```diff
@@ -10,24 +10,24 @@ pa.scheduler.home=.
 # you can use an absolute location or a relative location from scheduler home
 # pa.hsqldb.location=data/db
 
-# Scheduler REST url. If not defined, it is set automatically when starting the server.
+# Scheduler rest url. If not defined, it is set automatically when starting the server.
```
Comment (mboussaa) [inline src/docs/admin/references/properties/scheduler.properties:13]: REST => rest. Here as well if you want https://github.com/ow2-proactive/documentation/pull/729/files?short_path=ad0eb21#diff-607f3f73524204d4e30d7dfaa6bfc75152d69d7867d866dfc3a96d011644ef28L296

File: src/docs/admin/references/properties/scheduler.properties (hunk 1 above)
Comment (fviale) [reply src/docs/admin/references/properties/scheduler.properties:46]: this is downloaded automatically from the scheduling scheduler.properties file. Changes need to be done in scheduling.

File: src/docs/admin/references/properties/scheduler.properties (hunk 2 above)
Comment (fviale) [reply src/docs/admin/references/properties/scheduler.properties:13]: same remarks

File: src/docs/admin/references/properties/scheduler.properties [hunk 3]
```diff
@@ -10,24 +10,24 @@ pa.scheduler.home=.
...
 # Accessed inside workflows and tasks with the PA_SCHEDULER_REST_URL variable
 #pa.scheduler.rest.url=
 
-# Catalog REST url. If not defined, it is set automatically when starting the server.
+# Catalog rest url. If not defined, it is set automatically when starting the server.
 # Accessed inside workflows and tasks with the PA_CATALOG_REST_URL variable
 #pa.catalog.rest.url=
 
-# Service Automation REST url. If not defined, it is set automatically when starting the server.
+# Cloud Automation rest url. If not defined, it is set automatically when starting the server.
```
Comment (mboussaa) [inline src/docs/admin/references/properties/scheduler.properties:22]: ok for me if you prefer to use old naming on the server side. Which is normal as we didn't change microservice name

Comment (fviale) [reply src/docs/admin/references/properties/scheduler.properties:22]: same answer

File: src/docs/admin/references/properties/scheduler.properties (hunk 1 above)
Comment (mboussaa) [reply src/docs/admin/references/properties/scheduler.properties:46]: I mean change in the comment cloud automation => Cloud Automation, not the property

**Review by mboussaa — APPROVED** [review]

File: src/docs/admin/references/properties/scheduler.properties (hunk 1 above)
Comment (fviale) [reply src/docs/admin/references/properties/scheduler.properties:46]: I understand, but I cannot change it here. scheduler.properties is automatically downloaded when the project is built.

Comment (mboussaa) [reply src/docs/admin/references/properties/scheduler.properties:46]: ok

**Pull request merged**